liver_imaging_analysis/config/configs.json

```diff
@@ -47,16 +47,47 @@
         "sw_batch_size": 2,
         "roi_size": [192, 192, 32],
         "overlap": 0.25,
-        "train_transform": "3DUnet_transform",
-        "test_transform": "3DUnet_transform",
-        "post_transform": "3DUnet_transform"
+        "train_transform": "3d_ct_transform",
+        "test_transform": "3d_ct_transform",
+        "post_transform": "3d_ct_transform"
     },
     "save": {
         "volume_png_path": "Temp2D/volume/",
         "mask_png_path": "Temp2D/mask/",
-        "model_checkpoint": "model_checkpoint",
+        "model_checkpoint": "model_checkpoint",       
         "potential_checkpoint": "potential_checkpoint",
-        "tensorboard": "runs/"
+        "tensorboard": "runs/",
+        "output_folder":"output_folder"
     },
+
+    "test": {
+        "train_folder": "tests/testdata/data/resized_train_liver",
+        "test_folder": "tests/testdata/data/resized_liver",
+        "test_volume": "tests/testdata/data/resized_liver/volume/resized_liver.nii",
+        "test_label": "tests/testdata/data/resized_liver/mask/resized_mask.nii",
+        "reference_sliding_window": "tests/testdata/checkpoints/reference_liver_sw",
+        "liver_cp": "tests/testdata/checkpoints/liver_cp",
+        "lesion_cp": "tests/testdata/checkpoints/lesion_cp_tr",
+        "lobe_cp_sw": "tests/testdata/checkpoints/lobe_cp_sliding_window",
+        "liver_cp_sw": "tests/testdata/checkpoints/liver_sw",
+        "reference_lesion_cp": "tests/testdata/checkpoints/reference_lesion_cp",
+        "refrence_lesions_array": "tests/testdata/testcases/predicted_array_lesion.npy",
+        "reference_liver_2d_array": "tests/testdata/testcases/predicted_3d.npy",
+        "reference_liver_sw_array": "tests/testdata/testcases/predicted_sliding.npy",
+        "reference_lobe_array": "tests/testdata/testcases/predicted_lobe.npy",
+        "reference_gif": "tests/testdata/testcases/gif_frames",
+        "liver_slices": "tests/testdata/testcases/liverSlices",
+        "liver_png_slices": "tests/testdata/testcases/liverSlices_png",
+        "reference_batches": "tests/testdata/testcases/batches"
```

Comment:
Not sure what is this for, but whatever you're using this for, I don't think its right.

Reply:
These are old ones for the unit testing. As I remember they were already merged in main, but I don't know when or why they got removed.